lib/templates/baseLayout.mustache

```diff
@@ -472,9 +472,7 @@
                         {{#getGeneralErrorsGen}}
                             <tr>
                                 <td>{{operationInfo.operationId}}
-                                {{#coverageResultsForRendering}}
```

Comment:
Why is this tag removed?

Reply:
got  spec link data for render with operationId from getGeneralErrorsGen not from coverageResultsForRendering